src/braket/experimental/autoqasm/api.py

```diff
@@ -87,6 +90,30 @@ def gate(*args) -> Callable[[Any], None]:
     return _function_wrapper(args, _convert_gate)
 
 
+def pulse_sequence(*args, **kwargs):
+    if "implements" in kwargs:
+        assert len(args) == 0, "Cannot specify both `implements` and positional arguments."
+        return _calibration(**kwargs)
+    else:
+        return main(*args, **kwargs)
```

Comment:
Would it make more sense to treat this as a `subroutine`, so that the entry point always must use `@aq.main`? Or, I wonder if we just don't allow this mode at all for now, until we can enforce that it contains only pulses.

Reply:
The entry point of gate calibration is now simplified as @aq.gate_calibration. It only has one functionality: defining a gate calibration. The old `@pulse_sequence` does not add more functionality, as users can already create a purely pulse program with aq.main.